contracts/child/bor/StateSyncerVerifier.sol

```diff
@@ -25,6 +25,7 @@ contract StateSyncerVerifier is Ownable {
   function isOnlyStateSyncerContract() public view returns (bool) {
     return msg.sender == stateSyncer;
   }
+  // TODO: replace onlyOwner ownership with 0x1000
```

Comment:
Just having this comment here is not helping. These changes will need a change at a lot of places anyway to please remove this confusing comment.

Reply:
no, let's keep it here, at least I keep forgetting things so it'll help me remind that we haven't actually added validators 2/3 ownership yet.